test/integration/instance.test.js

```diff
@@ -898,6 +898,38 @@ describe(Support.getTestDialectTeaser('Instance'), function() {
       });
     });
 
+    it('handles an entry with primaryKey of zero', function() {
+      var username = 'user'
+        , newUsername = 'newUser'
+        , User2 = this.sequelize.define('User2',
+          {
+            id: {
+              type: DataTypes.INTEGER.UNSIGNED,
```

Comment:
I'll take that to mean I should leave the test as-is. ;)

Anyone else have thoughts on the code change?

> On May 29, 2015, at 3:30 PM, Ruben Bridgewater <notifications@github.com> wrote:
> 
> In test/integration/instance.test.js:
> 
> > @@ -898,6 +898,38 @@ describe(Support.getTestDialectTeaser('Instance'), function() {
> >        });
> >      });
> >  
> > +    it('handles an entry with primaryKey of zero', function() {
> > +      var username = 'user'
> > +        , newUsername = 'newUser'
> > +        , User2 = this.sequelize.define('User2',
> > +          {
> > +            id: {
> > +              type: DataTypes.INTEGER.UNSIGNED,
> Seems like you found a bug ^^. Mssql does not support unsigned integer. I'll fix that later.
> 
> —
> Reply to this email directly or view it on GitHub.
>